### PR TITLE
Retrieve bias metrics from thanos querier (#1472)

### PR DESCRIPTION
**Pull request by alexcreasy**
Closes: #1472 

## Description

* Adds backend endpoint for making range queries to thanos
* Updates Bias Prometheus queries to use this new endpoint

## How Has This Been Tested?

* Using a cluster with the latest version of trustyai: verify that model bias graphs correctly show data and that the Endpoint performance graph still works.

## Test Impact

Minor change of queries to service that as too many dependencies.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: backend/src/routes/api/prometheus/index.ts [hunk 1]
```diff
@@ -39,6 +40,19 @@ module.exports = async (fastify: KubeFastifyInstance) => {
     },
   );
 
+  fastify.post(
+    '/pvc-range',
```
Comment (andrewballantyne) [inline backend/src/routes/api/prometheus/index.ts:44]: are you sure this is `pvc` and not just data from Thanos?

Comment (andrewballantyne) [reply backend/src/routes/api/prometheus/index.ts:44]: Based on looking at the query -- you are not after PVC data...

PVC => Persistent Volume Claim; aka a cluster storage.

You'll need to name this as something else. Your data is just in Thanos / UWM (I think they are the same).

Can we rename `prometheusPVC` to `prometheusThanos` function and you call your endpoint something for your use case?

Comment (alexcreasy) [reply backend/src/routes/api/prometheus/index.ts:44]: Makes sense, can do!

**Review by DaoDaoNoCode — APPROVED** [review]
Tested, lgtm

**Review by andrewballantyne — APPROVED** [review]
lgtm from a code perspective.

Looks to be that Juntao tested it, so we can merge it I imagine.

**Pull request merged**